### ISSUE TITLE
Support for Dart-Sass

### ISSUE THREAD

**Issue by vikmn** [gruntjs/grunt-contrib-sass#239]
The readme mentions the requirement of ruby and sass for the purpose of this task. However it appears that the newer version of Sass is being moved away from ruby and uses dart. Also the newer version is available as an npm package @ github.com/sass/dart-sass. Will this task work out of the box without ruby if  the new package was to be used?

**Comment by nex3** [comment]
Dart Sass is in Release Candidate phase now, with a full release probably coming next week. I'd be happy to send a pull-request to add support for it if a maintainer can help me land it.

**Comment by laurenhamel** [comment]
I submitted a PR for a proposed upgrade to using Dart Sass. I left the integration tools (travis/appveyor) intact since you guys will likely need to review that stuff based on the proposed changes I'm suggesting. In the meantime, I've also created [`grunt-dart-sass`](https://github.com/laurenhamel/grunt-dart-sass), which uses the [Dart Sass JavaScript API](https://github.com/sass/dart-sass#javascript-api), for anyone who may land here looking for the same thing I was.

**Comment by ve3** [comment]
@laurenhamel It seems that this has not been merged? https://github.com/gruntjs/grunt-contrib-sass/pull/244

**Comment by laurenhamel** [comment]
@ve3, no, it was not. The PR I submitted (#244) was closed without review because of "too many changes," and it seems that this project is moving in the direction of an implementation-agnostic grunt task. If you're looking for a an immediate solution using Dart Sass, [refer to my previous comment](https://github.com/gruntjs/grunt-contrib-sass/issues/239#issuecomment-395429207).
